src/torrent.cpp

```diff
@@ -5664,99 +5675,66 @@ namespace {
 	void torrent::on_file_priority(disk_io_job const* j)
```

Comment:
is this the actual fix?

Reply:
Yes, it is in this function, also in `torrent::prioritize_files` and `torrent::set_file_priority`.